src/vm/comdelegate.cpp

```diff
@@ -2593,15 +2593,6 @@ BOOL COMDelegate::NeedsWrapperDelegate(MethodDesc* pTargetMD)
 {
     LIMITED_METHOD_CONTRACT;
 
-#ifdef _TARGET_ARM_
```

Comment:
I do not think it is ok to just return `false` for ARM from here. You should see crashes in delegate tests with this change. This change would need matching changes in `arm\virtualcallstubcpu.hpp` and JIT.